pronotepy/dataClasses.py

```diff
@@ -1219,3 +1219,131 @@ def students(self, period: Period = None) -> List[Student]:
                               {"classe": {"N": self.id, "G": 1}, "periode": {"N": period.id, "G": 1}})
         return [Student(self._client, j) for j in r["donneesSec"]["donnees"]["listeRessources"]["V"]]
+
+
+class Menu(Object):
+    """
+    Represents the menu of a meal
+
+    Attributes
+    ----------
+    id : str
+    self.name: Optional[str]
+    date : datetime.date
+        the date of the menu
+    is_lunch : bool
+        the menu is a lunch menu
+    is_dinner : bool
+        the menu is a dinner menu
+    first_meal : Optional[List[Food]]
+        food list of first meal
+    main_meal : Optional[List[Food]]
+        food list of main meal
+    side_meal : Optional[List[Food]]
+        food list of side meal
+    other_meal : Optional[List[Food]]
+        food list of other meal
+    cheese : Optional[List[Food]]
+        food list of cheese
+    dessert : Optional[List[Food]]
+        food list of dessert
+    """
+
+    class Food(Object):
+        """
+        Represents food of a menu
+
+        Attributes
+        ----------
+        id : str
+        name : str
+        labels : List[FoodLabel]
+        """
+
+        class FoodLabel(Object):
+            """
+            Represents the label of a food
+
+            Attributes
+            ----------
+            id : str
+            name: str
+            color: str
+            """
+
+            __slots__ = ["name", "id", "color", "_client"]
+
+            def __init__(self, client: _ClientBase, json_dict: dict) -> None:
+                super().__init__(json_dict)
+
+                self.id: str = self._resolver(str, "N")
+                self.name: str = self._resolver(str, "L")
+                self.color: str = self._resolver(str, 'couleur')
+
+                self._client = client
+
+                del self._resolver
+
+        __slots__ = ["name", "id", "labels", "_client"]
+
+        def __init__(self, client: _ClientBase, json_dict: dict) -> None:
+            super().__init__(json_dict)
+
+            self.id: str = self._resolver(str, "N")
+            self.name: str = self._resolver(str, "L")
+            self.labels: List[self.FoodLabel] = [self.FoodLabel(
+                client, label) for label in json_dict["listeLabelsAlimentaires"]["V"]]
+
+            self._client = client
+
+            del self._resolver
+
+    __slots__ = ["name", "id", "date", "is_lunch", "is_dinner", "first_meal",
+                 "main_meal", "side_meal", "other_meal", "cheese", "dessert", "_client"]
+
+    def __init__(self, client: _ClientBase, json_dict: dict) -> None:
+        super().__init__(json_dict)
+
+        self.id: str = self._resolver(str, "N")
+        self.name: Optional[str] = self._resolver(str, "L", strict=False)
+        self.date: datetime.date = self._resolver(Util.date_parse, 'Date', 'V')
+
+        self.is_lunch: bool = (self._resolver(int, 'G') == 0)
+        self.is_dinner: bool = (self._resolver(int, 'G') == 1)
```

Comment:
You don't need the parentheses:
```suggestion
        self.is_lunch: bool = self._resolver(int, 'G') == 0
        self.is_dinner: bool = self._resolver(int, 'G') == 1
```